src/translate.rs

```diff
@@ -364,107 +364,108 @@ impl<'a, 'tcx> TranslationContext<'a, 'tcx> {
         index_map: &HashMap<u32, DefId>,
         predicate: Predicate<'tcx>,
     ) -> Option<Predicate<'tcx>> {
+        use rustc_middle::ty::PredicateAtom;
```

Comment:
I guess this should eventually be moved in the `use` below...

Reply:
Yep, that's draft for now. I need to be extra careful around the predicates still, so I'll move the use statement around once I revisit the commit that introduced that and fix remaining tests.